apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TransactionTests.cs

```diff
@@ -159,6 +172,7 @@ private class HistoryEntry
         public async Task AbortedThrownCorrectly()
         {
             await WriteSampleRowsAsync();
+            LogPerformanceDataBeforeTest();
```

Comment:
It may be cleaner to have a disposable helper object which logs performance data in both its constructor and `Dispose` implementation. Then you can wrap all the tests in something like:
```csharp
using (new PerformanceLogHelper())
{
   ...
}
```

This would also prevent missing 'after' logging when returning early in one of the logged regions (although that seems unlikely for a test).